admin_app/src/app/content/page.tsx

```diff
@@ -109,38 +110,54 @@ const CardsUtilityStrip = ({
       sx={{ px: sizes.baseGap }}
     >
       <Layout.FlexBox
-        sx={{ width: { xs: "100%", md: "auto" } }}
+        sx={{ width: "auto" }}
         flexDirection={"row"}
+        alignItems={"center"}
       >
-        <Sort sx={{ display: { xs: "none", md: "flex" } }} />
+        <Sort sx={{ display: "flex" }} />
+        <Layout.Spacer horizontal multiplier={1} />
         <FormControl sx={{ width: "100%" }}>
           <InputLabel>Language</InputLabel>
           <Select
-            value={displayLanguage}
+            value={displayLanguage ? displayLanguage.language_name : ""}
+            value={displayLanguage ? displayLanguage.language_name : ""}
             label="Language"
-            onChange={({ target: { value } }) => onChangeDisplayLanguage(value)}
+            onChange={({ target }) => {
+              const selectedLanguage = languageOptions.find(
+                (lang) => lang.language_name === target.value,
+              );
+              if (selectedLanguage) {
+                onChangeDisplayLanguage(selectedLanguage);
+              }
+            }}
             sx={{
               backgroundColor: appColors.white,
             }}
           >
-            {loadingLanguages
-              ? <MenuItem value="">Loading...</MenuItem>
-              : languageOptions.map((language) => (
-                <MenuItem key={language.language_id} value={language.language_name}>{language.language_name}</MenuItem>
+            {loadingLanguages ? (
+              <MenuItem value="">Loading...</MenuItem>
```

Comment:
This menu item is to add a new language version to an existing content. Handles what happens when we click on the plus button on the `LanguageBar`